spec/models/question_spec.rb

```diff
@@ -1,5 +1,49 @@
 require 'rails_helper'
 
 RSpec.describe Question, type: :model do
+  let(:user) { build(:user) }
+  let(:question){create(:question, user: user)}
+
   it_behaves_like "a votable", :question_with_votes
+
+  describe "#time_updated" do
+    it "returns nil if time is not updated" do
+      expect(question.time_updated).to be nil
+    end
+
+    it "returns time difference if time is updated" do
+      question.created_at = DateTime.yesterday
+      expect(question.time_updated).to eq "less than a minute ago"
+    end
+  end
+
+    describe "#tags_to_a" do
+      it "is tagged" do
+        expect(question.tags_to_a).to eq []
+      end
+    end
```

Comment:
What do think; say we try this out with a question sample that has tags so as to check that it doesn't always return an empty array
 ```ruby
it 'returns empty array if question has no tag' do
  expect(question.tags.count).to eq 0
  expect(question.tags_to_a).to eq []
end

it "returns array contain tag names" do
   tag = create(:tag, name: "Zhishi")
   create(:question_resource_tag, tag: tag, question: question)
   expect(question.tags.count).to eq 1
   expect(question.tags_to_a).to eq ['Zhishi']
end
```
or something in that line